editor/gui/inspector_dock.cpp

```diff
@@ -211,6 +211,7 @@ void InspectorDock::draw(float deltaMilliseconds)
 						}
 					}
 				}
+				ImGui::Checkbox("Stop work on Pause", &m_OpenedScene->getScenePause());
```

Comment:
What does "Stop work on pause" mean?

Reply:
Stop the systems of that scene during `pause` .
Renaming it to